osu.Game/Graphics/UserInterface/FPSCounter.cs

```diff
@@ -159,6 +170,8 @@ protected override void Update()
         {
             base.Update();
 
+            mainContent.Width = Math.Max(mainContent.Width, counters.DrawWidth);
```

Comment:
That looks like basically `mainContent.AutoSizeAxes = Axes.X`? Unless you went through an issue with it?

Reply:
Note that it only increases

Reply:
Ah, nevermind.